### ISSUE TITLE
Task Distribution

### ISSUE THREAD

**Issue by reshnashrestha** [ubsuny/23-Homework4G1#12]
Hi @LinxuanHu and @Mosaddeq107, for this homework, I am hoping that I can take the algorithm task because I have done documentation and unit test already and now I am willing to do the algorithm. So let's define our task.

**Comment by Mosaddeq107** [comment]
Prof Tim said we don't need to develop any algorithm for this homework. We will use any algorithm from our previous two assignments. All we need to do is to make documentation, lint, and unit testing. Please, correct me if I'm wrong.

**Comment by Mosaddeq107** [comment]
@reshnashrestha

**Comment by reshnashrestha** [comment]
Yes @Mosaddeq107 thanks for clarifying. So I'll do linting. Is it fine?

**Comment by reshnashrestha** [comment]
Have you decided on the task distribution? @LinxuanHu @Mosaddeq107

**Comment by LinxuanHu** [comment]
I can make documentations

**Comment by reshnashrestha** [comment]
Sure

**Comment by reshnashrestha** [comment]
@Mosaddeq107 @LinxuanHu I have imported the hw3 algorithm we are going to work on and created the GitHub actions for it including the unit test and yml file for the unit test. I am hoping as @LinxuanHu voted for documentation, @Mosaddeq107 you can go for linting and pytest on the unit test
